src/features/search/components/__tests__/SearchResults.native.test.tsx

```diff
@@ -185,13 +185,17 @@ describe('SearchResults component', () => {
       expect(queryByTestId('typeButton')).toBeTruthy()
     })
 
-    it('should redirect to the general filters page when clicking on the type filter button', () => {
+    it('should redirect to the general filters page when clicking on the type filter button', async () => {
```

Comment:
Le nom de ton test n'est plus ok, ça serait plus :
`should open the type filter modal when clicking on the type filter button